kubernetes/chart/zulip/templates/NOTES.txt

```diff
@@ -0,0 +1,15 @@
+1. Zulip should now be available on:
+{{- if .Values.ingress.enabled }}
+{{- range .Values.ingress.hosts }}
+  http://{{ .host }}
+{{- end }}
+{{- else }}
+  export POD_NAME=$(kubectl get pods --namespace {{ .Release.Namespace }} -l "app={{ template "zulip.name" . }},release={{ .Release.Name }}" -o jsonpath="{.items[0].metadata.name}")
+  echo "Visit http://127.0.0.1:8080 to use your application"
+  kubectl port-forward $POD_NAME 8080:80
+{{- end }}
+
+2. To create a realm so you can sign in:
+
+  kubectl -n {{ .Release.Namespace }} exec -it $(kubectl get pods --namespace {{ .Release.Namespace }} -l "app={{ template "zulip.name" . }},release={{ .Release.Name }}" -o jsonpath="{.items[0].metadata.name}") -c zulip /home/zulip/deployments/current/manage.py generate_realm_creation_link
+
```

Comment:
I test-installed the chart, so far so good, but I couldn't succeed with creating a realm:

```
❯ kubectl -n zulip exec -it $(kubectl get pods --namespace zulip -l "app=zulip,release=zulip" -o jsonpath="{.items[0].metadata.name}") -c zulip /home/zulip/deployments/current/manage.py generate_realm_creation_link

error: error executing jsonpath "{.items[0].metadata.name}": Error executing template: array index out of bounds: index 0, length 0. Printing more information for debugging the template:
	template was:
		{.items[0].metadata.name}
	object given to jsonpath engine was:
		map[string]interface {}{"apiVersion":"v1", "items":[]interface {}{}, "kind":"List", "metadata":map[string]interface {}{"resourceVersion":"", "selfLink":""}}


kubectl exec [POD] [COMMAND] is DEPRECATED and will be removed in a future version. Use kubectl exec [POD] -- [COMMAND] instead.
error: arguments in resource/name form may not have more than one slash
```